config/configuration.go

```diff
@@ -16,162 +16,50 @@ package config
 
 import (
 	"fmt"
+	"log"
 	"os"
+	"strings"
 
-	"github.com/spf13/afero"
-	"gopkg.in/yaml.v2"
+	"github.com/BurntSushi/toml"
+	"github.com/spf13/viper"
 )
 
-var AppFs = afero.NewOsFs()
+var confDir = ".edgex-cli"
```

Comment:
If we assume for the moment that we are running out of a git repo, then .edge-cli would be in our git directory. That simplifies a lot of things. The whole home directory hidden file is a complication to support EdgeX-cli in our command path and then needing a fixed location for the config file but that becomes moot when we allow specifying a config file.